### PR TITLE
Add authentication for Adobe HDS streaming.

### PR DESCRIPTION
**Pull request by tingulfsen**
This adds support for Adobe HDS streaming, which we want to try out for MP3 content.

### CODE REVIEW

**Comment by jmartin-sul** [comment]
👍

File: src/edu/stanford/dlss/wowza/SulWowza.java [hunk 1]
```diff
@@ -82,6 +83,22 @@ public void onHTTPCupertinoStreamingSessionCreate(HTTPStreamerSessionCupertino h
         }
     }
 
+    /** Invoked when an Adobe HDS streaming session is created (known as San Jose Streaming in Wowza parlance).
+     *  This is an HTTP-based (port 80) streaming protocol for Flash. We currently use this for streaming MP3
+     *  files, since there are no current HLS players that support MP3 across all browsers. */
+    public void onHTTPSanJoseStreamingSessionCreate(HTTPStreamerSessionSanJose httpSession)
+    {
+        if(invalidConfiguration)
```
Comment (ndushay) [inline src/edu/stanford/dlss/wowza/SulWowza.java]: space b/t "if" and "(" like other stuff?

**Comment by ndushay** [comment]
Just a quick fix on spacing to match the rest of the file.  Also, I think we are at our max for method with essentially the same content, but "good enough for shameless green" applies, I think.

**Comment by tingulfsen** [comment]
Thanks. I agree that some refactoring should be done the next time we touch this code.

**Pull request merged**